### PR TITLE
Review wanted: Weather app, Mathias

### PR DESCRIPTION
**Pull request by Nesiht**
Link to my deployed project:
[Link](https://goofy-torvalds-e1df57.netlify.com/)

My reflections on how this week's project turned out:
Would like to spend more time on the design and a feature for the user to search for a city.
Tried to create it but never got it to work correctly with the forecast :/

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

**Comment by ericaglimsholt** [comment]
Hi! Is this project done and ready for a review? If yes - please update the PR title to "review wanted:" before your title you have now (you can edit your PR here on github). This is for the projects that's done before the deadline. If you change it to "review wanted" we will try and give you a review as soon as possible.

**Comment by Nesiht** [comment]
Updated the title.

File: code/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        <article class="small-card">
+          <p class="icon sun">Sunrise</p>
+          <p class="icon rise" id="sunrise"></p>
+          <p class="icon set" id="sunset"></p>
+          <p class="icon sun">Sunset</p>
+        </article>
+    </article>
+
+    <article class="big-card" id="updateSearch">
+      <h1>5 day forecast</h1>
```
Comment (ericaglimsholt) [inline code/index.html]: Try to use only one `<h1>` per page!

File: code/script.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+const description = document.getElementById("description")
+const sunrise = document.getElementById("sunrise")
+const sunset = document.getElementById("sunset")
+const forecast = document.getElementById("forecast")
+const img = document.getElementById("show-img")
+const weekday = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]
+
+
+const weatherKey = "3c481a17ec1c4b275eed746ad29d58b1" 
+const stockholm = "2673730"
```
Comment (ericaglimsholt) [inline code/script.js]: Suggestion:
```suggestion
const stockholmID = "2673730"
```

File: code/script.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+const units = "metric"
+const weatherUrl = `https://api.openweathermap.org/data/2.5/forecast?id=${stockholm}&units=${units}&appid=${weatherKey}`
+console.log(weatherUrl)
```
Comment (ericaglimsholt) [inline code/script.js]:
```suggestion
```

**Pull request closed without merging**